lib/msal-core/samples/VanillaJSTestApp/server.js

```diff
@@ -20,7 +20,7 @@ app.use("/bower_components", express.static(path.join(__dirname, 'bower_componen
 
 // Set up our one route to the index.html file.
 app.get('*', function (req, res) {
-    res.sendFile(path.join(__dirname + '/index_LoginRedirect.html'));
+    res.sendFile(path.join(__dirname + '/index_LoginPopup.html'));
```

Comment:
dont need to do now, but when we work on samples, might be good to have different express routes for each scenario

Reply:
Samples will have a different PR.